src/tasks/daily/daily_battle_mixin.py

```
@@ -416,7 +416,9 @@ def battle_recycle(self, left_ticket, stage_name, category_name, enter_str, no_b
                 self.wait_click_ocr(match=re.compile("重新挑战"), box=self.box.bottom_left, log=True, time_out=5,
                                     after_sleep=2, recheck_time=1)
             else:
-                self.wait_click_ocr(match=re.compile(enter_str), time_out=10, after_sleep=2, box=self.box.bottom_right,
+                # 根据 enter_str 决定点击位置，如果是「挑战」，则点击右下四分之一区域，避免误触
```

Ruff (0.15.11)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)